src/mainwindow.cpp

```diff
@@ -584,7 +584,7 @@ void MainWindow::onActionFinished(FrAction action, ArchiverError err) {
         break;
     case FR_ACTION_TESTING_ARCHIVE:            /* testing the archive integrity */
         if(!err.hasError()) {
-            QMessageBox::information(this, tr("Success"), tr("No errors are found."));
+            QMessageBox::information(this, tr("Successful"), tr("No errors have been found."));
```

Comment:
I believe as a window title, 'success' is actually better here, if using just one word.

A successful something results in a success ("Testing Successful" for example).

Reply:
I believe this too :)